client/state/posts/utils.js

```diff
@@ -31,7 +31,9 @@ import stripHtml from 'lib/post-normalizer/rule-strip-html';
  * Constants
  */
 
-const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/,
+	REGEXP_EMPTY_CONTENT = /^<p>(<br[^>]*>|&nbsp;|\s)*<\/p>$/,
+	CONTENT_LENGTH_ASSUME_SET = 50;
```

Comment:
I prefer also one const per variable, but I found inconsistencies on the codebase, maybe we should add this ESLint rule as a warning 👍